src/components/DisplayComponent.js

```diff
@@ -250,11 +251,11 @@ class DisplayComponent extends Component {
     });
     const { ExportCSVButton } = CSVExport;
     return (
-      <div id="mainDisplay">
+      <span id="mainDisplay">
```

Comment:
Container should probably stay as `div`, I thought the change to `span` on line 258 is already enough to make the buttons on the same line?

Reply:
Can export button be aligned to right?